optaplanner-examples/src/main/java/org/optaplanner/examples/conferencescheduling/persistence/ConferenceSchedulingXlsxFileIO.java

```diff
@@ -1013,10 +1013,14 @@ private void writeTalkList() {
                 nextCell().setCellValue(talk.getFavoriteCount());
                 nextCell().setCellValue(talk.getCrowdControlRisk());
                 nextCell(talk.isPinnedByUser() ? pinnedStyle : defaultStyle).setCellValue(talk.isPinnedByUser());
-                nextCell().setCellValue(talk.getTimeslot() == null ? "" : DAY_FORMATTER.format(talk.getTimeslot().getDate()));
-                nextCell().setCellValue(talk.getTimeslot() == null ? "" : TIME_FORMATTER.format(talk.getTimeslot().getStartDateTime()));
-                nextCell().setCellValue(talk.getTimeslot() == null ? "" : TIME_FORMATTER.format(talk.getTimeslot().getEndDateTime()));
-                nextCell().setCellValue(talk.getRoom() == null ? "" : talk.getRoom().getName());
+                nextCell(talk.getTimeslot().equals(talk.getPublishedTimeslot()) ? defaultStyle : hardPenaltyStyle)
```

Comment:
hardPenaltyStyle? The timeslot one is a medium constraint and the room one is a soft constraint.

Let's use a different color (= style) altogether, maybe something like republishedStyle? Not green (= good), not yellow/orange/red (used by hard soft) not purple (used by pinned). So how about blue or pink?

Reply:
Blue implies something is ok (it's close to green), so I think pink is better (or magenta maybe?)

Reply:
Sounds good